### ISSUE TITLE
Setting up an Arch Linux repository using GitHub

### ISSUE THREAD

**Issue by fusion809** [tensor5/arch-atom#16]
Hi,

I've forked your repository (it's not listed as a fork by accident as I deleted the original forked repository from my GitHub account and then created an arch-atom repository and committed my local copy of the original forked repo to it) [fusion809/arch-atom](https://github.com/fusion809/arch-atom) so I can provide customized Atom packages. Now I'm mentioning this because I've learnt something you may be interested in, as it may save you some money on hosting your packages at noaxiom.org. 

If you tag a release (go to https://github.com/tensor5/arch-atom/releases to do this, if you aren't comfortable tagging from the command-line) and attach your binaries to your tag, along with your database files (i.e., your `atom.db`, `atom.files`, `atom.db.tar.gz` and `atom.files.tar.gz` files). Then the `Server` field for one's `/etc/pacman.conf` file would be `https://github.com/tensor5/arch-atom/releases/download/<TAG>` where `<TAG>` is, of course, your tag. For example my first tag was `20160705` so in my `/etc/pacman.conf` file my repository looks like:

```bash
[atom-bleeding]
Server = https://github.com/fusion809/arch-atom/releases/download/20160705/
SigLevel = Optional TrustAll
```

granted new tags mean new `Server` values, although you can use just the one tag for all binary releases if you want, but using new tags seems like a better practice. 

Hope this helps,
Brenton

**Comment by tensor5** [comment]
Thank you for the tip; I keep the noaxiom.org server and domain also for other stuff, including the Arch repository [[haskell-happstack]](https://github.com/tensor5/haskell-happstack).

I looked at your fork, and I noticed that one major feature is a custom package choice. I would propose to investigate the possibility to make Arch packages of individual Atom packages that could be installed at system level, not in `~/.atom`. That way we could share packages, and you wouldn't need to rebuild Atom every time a new package is released. I'm not sure if it's easy to do it though, or worth the effort.
